### DISCUSSION TITLE
How to create form to accept an array of items

### DISCUSSION THREAD

**Discussion by micebrain** [Budibase/budibase#3394] [Help] [answered]
Hello!
I am trying to figure out which combination of components I could use in a form for a data entry that accept a list of items. The data is then use to post to the API with request body that look something like:

```
{
    "backup_name": "test",
    "host": "hostname",
    "folders" : [
        {"source": "...", "destination": "..." },
        {"source": "...", "destination": "..." },
    ]
}
```

The users should be able to enter a list of folders, for example like having editable "virtual rows" not connected to a table.
I look at the Repeater component, but couldn't figure out how to use it for this purpose.
Would appreciate if someone has an idea to this.

**Comment by mjashanks** [comment, marked as answer]
Hi @micebrain - this is not possible at the moment. However, we are currently looking into this: https://github.com/Budibase/budibase/issues/3381 - which I think could solve this for you.

**Reply by micebrain** [reply]
Okay, thanks good to know!
